src-docs/src/views/accordion/accordion_example.js

```diff
@@ -16,6 +16,23 @@ import {
 import Accordion from './accordion';
 const accordionSource = require('!!raw-loader!./accordion');
 const accordionHtml = renderToHtml(Accordion);
+const accordionSnippet = `<EuiAccordion
+  id="accordion1"
```

Comment:
Also, we should consider any time there's an `id` or `data-test-subj` or some other attribute/prop that should be unique to the consumer's implementation, to not set it up as a normal string, but as if it's referencing a JS variable so they'll get compile warnings for them to create this string.
```suggestion
  id={idOfAccordion}
```